app/controllers/insured/employee_roles_controller.rb

```diff
@@ -23,17 +23,20 @@ def search
 
   def match
     @no_save_button = true
-    @person_params = params.require(:person).permit(person_parameters_list).merge({:user_id => current_user.id})
+    session[:invalid_match_attempts] ||= 0
+    @person_params = params.require(:person).permit(person_parameters_list).merge({:user_id => current_user.id, :invalid_match_attempts => session[:invalid_match_attempts]})
     @person_params.merge(no_ssn: params.dig(:person, :no_ssn)) if params.dig(:person, :no_ssn)
     @person_params.merge(:dob => params.dig(:jq_datepicker_ignore_person, :dob)) if params.dig(:jq_datepicker_ignore_person, :dob)
+    @captcha_required = false
```

Comment:
if we deprecated the recaptcha code, do we still need this variable ?

Reply:
good catch, no we do not need this variable.